### DISCUSSION TITLE
upload  file issue

### DISCUSSION THREAD

**Discussion by Ali-kheiraldine** [SiEPIC/openEBL-2024-02#66] [Q&A]
Hello,
 I uploaded the gds file to the submissions directory and followed all the steps (...green check mark,ok) but at the end I don't see my design included in the merged design.
Thanks

**Comment by Sahar-Safarloo** [comment]
I have the same problem

**Reply by lukasc-ubc** [reply]
There were some issues that were resolved in the past few days. Could you please check now? I assume your designs were already merged?

**Reply by Sahar-Safarloo** [reply]
Yes, they were merged. My comment was not up to date. Thank you.

**Reply by Ali-kheiraldine** [reply]
Yes my design has already merged, Thanks
